nilearn/interfaces/fmriprep/load_confounds.py

```diff
@@ -260,6 +264,21 @@ def load_confounds(
           `~desc-smoothAROMAnonaggr_bold.nii.gz`.
         - "basic": use noise independent components only.
 
+    tedana : :obj:`str`, default="aggressive"
+
+        - "aggressive": use :term:`Tedana` tedana optimaly combined
```

Comment:
if you use :term:`Tedana` then you must add it to the glossary,
otherwise you get a failure in the doc build

/home/runner/work/nilearn/nilearn/.tox/doc/lib/python3.9/site-packages/nilearn/interfaces/fmriprep/load_confounds.py:docstring of nilearn.interfaces.fmriprep.load_confounds.load_confounds:147: WARNING: term not in glossary: 'Tedana'

https://github.com/nilearn/nilearn/actions/runs/16122467668/job/45504792675?pr=5410#step:13:1539

It should be added here: https://github.com/nilearn/nilearn/blob/347557ec24fe17ac9060ab3c28a48eff597ce2a6/doc/glossary.rst?plain=1#L302